metrics/traefik.go

```diff
@@ -76,146 +87,164 @@ func NewTraefikMetricProcessor(config []common.RulesConfig) (*TraefikMetricProce
 	return &mp, nil
 }
 
-func (mp TraefikMetricProcessor) getMetrics(logEntry model.TraefikLog, values map[string]string) (map[string]interface{}, error) {
-	m := map[string]interface{}{}
-
-	// status codes
-	integer, err := strconv.ParseInt(values["status"], 10, 64)
+func parseCommonLog(entry model.LogEntry) (map[string]interface{}, error) {
+	matches := ApacheCombinedLogRegex.FindStringSubmatch(entry.Log)
 
-	if err != nil {
-		return nil, err
+	if len(matches) != 15 {
```

Comment:
`15` looks like a good candidate to move to a const

Reply:
5f45fef777762f9bf3c2828ed2e84a70e9c7e878